.github/workflows/odis-loadtest-container.yml

```diff
@@ -42,3 +44,17 @@
         context: .
         file: dockerfiles/phone-number-privacy/Dockerfile-loadtest
         trivy: true
+
+    odis-loadtest-tag:
+      uses: celo-org/reusable-workflows/.github/workflows/container-cicd.yaml@v1.12.0
```

Comment:
Same as previous.
Also why different versions here?

Reply:
Testing latest commit [here](https://github.com/celo-org/social-connect/pull/73). BTW the `platforms` default doesn't seem to work